src/Blocks/continuous.jl

```diff
@@ -57,7 +66,7 @@ A smaller `T` leads to a more ideal approximation of the derivative.
 @mtkmodel Derivative begin
     @extend u, y = siso = SISO()
     @variables begin
-        x(t) = 0.0, [description = "Derivative-filter state"]
+        x(t), [description = "Derivative-filter state", guess = 0.0]
```

Comment:
Same here, there is no guessing involved, a default is much better.

Reply:
Resolved with: https://github.com/SciML/ModelingToolkitStandardLibrary.jl/pull/274/commits/3a7ccfbd43de07b1f1de3f81dc1b86e173146eac